### DISCUSSION TITLE
Change a repeater subnet

### DISCUSSION THREAD

**Discussion by dementeb** [Mygod/VPNHotspot#577] [General]
Hello

Is it theoretically possible to use a different subnet for the repeater, other than 192.168.49.0/24?

I connect 2 phones, the second one also distributes the Internet - and this chain does not work... I suspect that this is due to the fact that the second phone receives an IP in the network 192.168.49.0, which it is also trying to distribute.
wifi -> phone 1 repeater mode -> phone 2 repeater mode -> client

I can't use hotspot mode on 1 phone because it turns off WiFi...

I read in a closed issue that it is impossible to change the subnet in hotspot mode. Perhaps there is no such limitation in repeater mode?

Thank you

**Comment by Mygod** [comment]
This is hardcoded in `WifiServiceImpl` so there is no plan to support this.

**Reply by dementeb** [reply]
Thank you.
